src/classes/CW_CampaignSitesTriggerHandlerTest.cls

```diff
@@ -0,0 +1,68 @@
+@isTest
```

Comment:
Style issue: classes and triggers should use the "UpperCamelCase" notation. Suggestion: rename accordingly.